client/src/Utils/vendor/useDuplicateContactId.ts

```diff
@@ -1,14 +1,48 @@
+import { useSelector } from 'react-redux';
+
+import StoreStateType from 'redux/storeStateType';
 import logger from 'logger/logger';
 import { Service, Severity } from 'models/Logger';
 import useCustomSwal from 'commons/CustomSwal/useCustomSwal';
 
 export const duplicateIdsErrorMsg = 'found duplicate ids';
 
+export interface IdToCheck {
+    id?: string;
+    serialId?: number 
+}
+
 const useDuplicateContactId = () => {
     
-    const {alertWarning} = useCustomSwal();
+    const { alertWarning } = useCustomSwal();
+    const epidemiologyNumber = useSelector<StoreStateType, number>(state => state.investigation.epidemiologyNumber);
+    const userId = useSelector<StoreStateType, string>(state => state.user.id);
+
```

Comment:
The difference between the functions' goals isnt clear - maybe change the first function's name.
In addition, why checking distinct ids only on the second one?

Reply:
fixed